### PR TITLE
E-985 fix: order for show pending screen

### PR DESCRIPTION
**Pull request by dudiq**

### CODE REVIEW

**Comment by dudiq** [comment]
@lamartire ready for review

File: src/components/modules/document/DocSingleCreation/modules/UploadStatus/UploadStatus.interactor.vue [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  <upload-status-layout
+    :is-pending="isPending"
+    :is-verified="isVerified"
+    @continue="onContinue"
+    @create="onCreate"
+  />
+</template>
+
+<script>
+import UploadStatusLayout from '@/components/modules/document/common/UploadStatusLayout';
```
Comment (lamartire) [inline src/components/modules/document/DocSingleCreation/modules/UploadStatus/UploadStatus.interactor.vue:11]: Why you are using `Layout` here, not on `view` layer?

File: src/components/modules/document/DocSingleCreation/modules/UploadStatus/UploadStatus.interactor.vue [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    onContinue() {
+      this.$emit('continue');
+    },
+
+    onCreate() {
+      this.$emit('create');
+    },
+  },
+
+  async mounted() {
```
Comment (lamartire) [inline src/components/modules/document/DocSingleCreation/modules/UploadStatus/UploadStatus.interactor.vue:47]: Can we move all this logic to method?

File: src/components/modules/document/DocSingleCreation/modules/UploadStatus/UploadStatus.interactor.vue (hunk 1 above)
Comment (dudiq) [reply src/components/modules/document/DocSingleCreation/modules/UploadStatus/UploadStatus.interactor.vue:11]: what will be in `view` layer?

**Comment by ifiri** [comment]
@kazurus approved. In vault nothing broken as well.

**Pull request merged**